src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,51 @@
-export const Tabs = () => {};
+import { SyntheticEvent } from 'react';
+import { Tab, TabsProps } from '../../types/tab';
```

Comment:
not fixed
<img width="624" alt="image" src="https://user-images.githubusercontent.com/102411130/233003639-4054b98d-c20f-40de-adbb-aa8ef63ba69b.png">

Reply:
I don't get this problem any longer
What should I fix here?

Reply:
<img width="1282" alt="image" src="https://user-images.githubusercontent.com/101051619/233086372-61eb7bd4-7d61-41e7-8444-b5aa734e7164.png">
these are the only 2 tests that fail

Reply:
295416aece5f61e4b8e7b05a04ddcfd3b5cbcec6

Reply:
add types/tabs file to github